packages/ui-react/src/components/HeroShelf/HeroSwipeSlider.tsx

```diff
@@ -73,18 +88,17 @@ const HeroShelfMetadataMobile = ({ item, leftItem, rightItem, playlistId, loadin
 
   useLayoutEffect(() => {
     if (!containerRef.current) return;
-
-    if (direction === 'left') {
+    if (isSwipeAnimation && direction === 'left') {
       containerRef.current.style.transition = 'transform 0.2s ease-out';
       containerRef.current.style.transform = `translateX(100%)`;
-    } else if (direction === 'right') {
+    } else if (isSwipeAnimation && direction === 'right') {
```

Comment:
Would it look nicer for tablets and larger screen sizes to decrease the left/right offset and use opacity as well? It works great now, but the slide animation distance is pretty big (opinionated ofc)

Reply:
I think you mean the animation after a swipe gesture (because the normal slide hasn't changed). The thing is: when you drag the metadata over the screen, then let go, you want the gesture to be continued similarly. So I cannot really change the opacity, you expect it to be opaque. Same for the offset, it comes from outside of the screen. I could only update the transition perhaps?